op-acceptance-tests/tests/interop/loadtest/interop_load_test.go

```diff
@@ -68,19 +68,19 @@ func TestLoad(gt *testing.T) {
 }
 
 func SpamInteropTxs(t devtest.T, numInitTxs uint64, source *L2, dest *L2, supervisor *dsl.Supervisor) {
-	var wg sync.WaitGroup
-	defer wg.Wait()
+	var relayWg sync.WaitGroup
+	defer relayWg.Wait()
 	msgsCh := make(chan []types.Message, 100)
```

Comment:
Would be good to monitor the utilization of this buffer, if it's ever full that suggests there's a performance bottleneck in the relayer side.